docs/release_notes.md

```diff
@@ -6,62 +6,40 @@ Release Notes
 
 ### ⚠ BREAKING CHANGES
 
-* Replace `HLLCardinality` with `ApproxCountDistinct` and `CMSMedian` with `ApproxMedian` operations.
-* **backends:** The dtype of returned execution results now more closely matches that of the ibis expression's type. Downstream code may need to be adjusted.
-* The `JSONB` type is replaced by the `JSON` type.
-* **dev-deps:** Expression types have been removed from `ibis.expr.api`. Use `import ibis.expr.types as ir` to access these types.
-* **ir:** removed @immutable_property decorator, use @attribute.default instead
-* **timestamps:** The `timezone` argument to `to_timestamp` is gone. This was only supported in the BigQuery backend. Append `%Z` to the format string and the desired time zone to the input column if necessary.
-* **deps:** Ibis now supports at minimum duckdb 0.3.3. Please upgrade your duckdb install as needed.
-* Previously `ibis.connect` would return a `Table` object
-when calling `connect` on a parquet/csv file. This now returns a backend
-containing a single table created from that file. When possible users
-may use `ibis.read` instead to read files into ibis tables.
-* The `closed` argument no longer exists because it never had any effect. Remove it from your `histogram` method calls.
-* The Pandas and Dask backends now interpret casting ints
-to/from timestamps as seconds since the unix epoch, matching other
-backends.
-* **datafusion:** `register_csv` and `register_parquet` are removed.
-Pass filename to `register` method instead.
-
-This also adds in a general backend test for the `register` API so we
-can make sure things are consistent, and fixes a small bug with the
-dispatch in polars (and datafusion which I copied) that raises a
-recursion error if you pass in garbage.
-
-Update ibis/backends/tests/test_client.py
-
-Co-authored-by: Phillip Cloud <417981+cpcloud@users.noreply.github.com>
-* **ir:** NodeList and ir.List are removed. Use tuples instead.
+* functions, methods and classes marked as deprecated are removed now
```

Comment:
Does all of this need to be fixed? Can you just fix the datafusion text and leave the rest alone?

Reply:
The idea is to have the breaking change message match the release notes exactly, and improve over time. Rewriting history should be a last resort, in the case the docs are unreadable without the hand-editing so we have to do a bit of rewriting.

Reply:
It looks fine just fixing that bit:

![image](https://user-images.githubusercontent.com/417981/211550693-7992a6db-ebb7-4d0f-984e-9b17e977d165.png)

Reply:
Nevermind, this is fine as is.